### DISCUSSION TITLE
Changing the global text alignment

### DISCUSSION THREAD

**Discussion by mcgregorian1** [greenelab/lab-website-template#253] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://deadlabcary.github.io/lab-website/media/

### Version of Lab Website Template you are using

1.2.1

### Description

I'm trying to figure out how to make the text of different items (e.g. cards, home page) be something other than justified alignment (e.g. left, right, etc). I found in the `card.scss` file there are parameters for `align-items` and `justify-content`, but I'm not sure what the acceptable inputs are nor how they affect the site. I checked the documentation but couldn't find any specifics.

I would ultimately like to change this for the whole website, but the card example is emblematic of the issue.

**Comment by vincerubinetti** [comment]
There's not really a way to change the alignment of everything on the site, nor would that be advisable.

You'll need to have some [CSS knowledge](https://greene-lab.gitbook.io/lab-website-template-docs/advanced/background-knowledge) to edit the `_styles` files. For example, `justify-content` can be set to `flex-start`/`center`/`flex-end` and more. That property is how you set horizontal alignment in flex layouts, unless `flex-direction` is `column`, in which case it'd be `align-items` instead. 

Then `text-align` can be used to set text alignment within a block context. If you change this in `body.scss`, many (but not all) things will inherit from that (it can still be overridden, and flex layouts will still be whatever they're set to).

If you want help making a particular element look the way you want, please post a screenshot.

**Reply by mcgregorian1** [reply]
> There's not really a way to change the alignment of everything on the site, nor would that be advisable.

Right! My bad for the lack of clarity, I was mainly referring to wanting to change the text to not be as spaced or stretched out as it currently is on the repo page link I shared in the post (i.e. not justified alignment as in Microsoft Word). 

With regard to `text-align` vs `justify-content`, which of those changes the inter-text spacing? Based on this [text explainer](https://www.w3schools.com/css/css_text_align.asp) and this [justify-content explainer](https://www.w3schools.com/cssref/css3_pr_justify-content.php), it appears that it should be the former, but the `text-align` variable in `body.scss` is already set to center (not justify) and it is set to left in `card.scss`. However, I see the same justify behavior when the site is rendered. Do you know why?

**Reply by vincerubinetti** [reply, marked as answer]
You would be looking for anywhere in the code that `text-align: justify;`. Just ctrl+f your repo for that string and replace it with `text-align: left;`. In particular, you are probably noticing the one on `p` (paragraph) elements, which appear all over the template, including the card component:

<img width="200" alt="Screenshot 2024-04-17 at 9 35 12 AM" src="https://github.com/greenelab/lab-website-template/assets/8326331/e096c697-004d-4f79-ba25-63da2fb839ad">

**Reply by mcgregorian1** [reply]
Perfect that's the parameter I was looking for, thank you!
